readme.md

```diff
@@ -44,10 +44,11 @@ Heads up for Windows + Vagrant users: Start by applying the fix in [issue #61](h
 ```bash
 composer create-project jadjoubran/laravel5-angular-material-starter --prefer-dist
 cd laravel5-angular-material-starter
-#fix database credentials in .env
+#fix database credentials, add API_API_PREFIX and API_DOMAIN entries to .env
```

Comment:
Hello, and thanks for the clarifications!

I thought the composer install would do the same as a clone from Github.
Sorry about that! I'll have to read the instructions myself before editing
them, next time :)
On Nov 25, 2015 1:48 AM, "Jad Joubran" <notifications@github.com> wrote:

> In readme.md
> <https://github.com/jadjoubran/laravel5-angular-material-starter/pull/108#discussion_r45846697>
> :
>
> > @@ -44,10 +44,11 @@ Heads up for Windows + Vagrant users: Start by applying the fix in [issue #61](h
> >  ```bash
> >  composer create-project jadjoubran/laravel5-angular-material-starter --prefer-dist
> >  cd laravel5-angular-material-starter
> > -#fix database credentials in .env
> > +#fix database credentials, add API_API_PREFIX and API_DOMAIN entries to .env
>
>
>    - You can't add API_PREFIX & API_DOMAIN
>    - when you install this starter project, it copies the env.example to
>    .env which has a defualt value for API_PREFIX so usually you don't have to
>    update/change these unless you want to customize it yourself. But that
>    would break the default Restangular configuration, so I don't think it's
>    advised.. Let me know what you think
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jadjoubran/laravel5-angular-material-starter/pull/108/files#r45846697>
> .
>